### ISSUE TITLE
Question about saving the model weights in int

### ISSUE THREAD

**Issue by reiinakano** [worldmodels/worldmodels.github.io#16]
I've been playing around a bit with the world models code and using it in my own applications, and I noticed that your model saving code converts the float weights to integers by multiplying by 10000. Do you have any special reason for saving to ints and the 10000 value? I find that occasionally, my model screws up when I reload it from the saved weights. I'm suspecting it's because of this loss in precision but I'm not 100% sure. Just wondering if you've experienced any weirdness from this in the past.

**Comment by hardmaru** [comment]
To decrease the actual size of the model served over the web, I quantized the weights to 4 significant digits by dumping the values originally to a json INT16 array (that is base64 coded). When I load them to tensorflow.js (well, it's a really old version of it when it was still called deeplearn.js), I converted it back to floats by dividing each value by 10000.

I think with the newer tools available in tf.js, there are now much better ways to serve models efficiently over the web. I think there are plans to have these quantization schemes in the library.

**Comment by reiinakano** [comment]
Ah, that makes sense. I will stick with the original floats for stopping and restarting long running training then. Thanks!

**Comment by hardmaru** [comment]
If you are looking to train these models I would recommend training the
python-based versions first on your environment since they are already
known to work on the 2 environments described in the paper. Good luck!
